packages/app/cypress/e2e/studio/studio.cy.ts

```diff
@@ -33,6 +33,14 @@ describe('Cypress Studio', () => {
       })
     })
 
+    it('displays Studio header', () => {
+      launchStudio({ enableCloudStudio: true })
+
+      cy.viewport(1500, 1000)
+      cy.get('data-cy="studio-toolbar"').should('be.visible')
```

Comment:
does this work? Or does it need to be:
```suggestion
      cy.get('[data-cy="studio-toolbar"]').should('be.visible')
```